exercises/roman-numerals/test/Tests.hs

```diff
@@ -17,66 +17,85 @@ specs = describe "roman-numerals" $
 
     test Case{..} = it explanation assertion
       where
-        explanation = show number
+        explanation = show number ++ ": " ++ description
```

Comment:
like "27: 20 is two X's"

I wanted to keep the actual number being tested because in some cases the number in the description is not the input. Thought it might have been confusing to have the test output "20 is two X's" when the input is 27